experimental/Schemes/ProjectiveModules.jl

```diff
@@ -178,63 +182,68 @@ function _is_projective_without_denominators(A::MatElem; unit::RingElem=one(base
   powers = [k for (_, _, k) in sub_results]
   # Find the coefficients cᵣ so that ∑ᵣ cᵣ ⋅ uᵣᵏ⁽ʳ⁾ = 1 ∈ R.
   # TODO: This can be optimized
-  c = coordinates(one(R), ideal(R, [u^(k+1) for ((u, _, _), k) in zip(entry_list, powers)]))
+  if task == :with_projector
+    c = coordinates(one(R), ideal(R, [u^(k+1) for ((u, _, _), k) in zip(entry_list, powers)]))
 
-  result = zero(matrix_space(R, n, n))
-  for ((u, i, j), (_, Q, k), (Rloc, inc), lambda) in zip(entry_list, sub_results, sub_localizations, c)
-    # Lift the matrix Q to an n×n-matrix over R
-    Qinc = zero(matrix_space(R, n, n))
-    for r in 1:j-1
-      for s in 1:j-1
-        Qinc[r, s] = preimage(inc, Q[r, s])
-      end
-      for s in j+1:n
-        Qinc[r, s] = preimage(inc, Q[r, s-1])
+    result = zero(matrix_space(R, n, n))
```

Comment:
Rule of thumb: `matrix_space` really only exists to have a parent for matrices, there is rarely a valid reason to use it. E.g. here it's better to do this:
```suggestion
    result = zero_matrix(R, n, n)
```
Same elsewhere.

This is no reason to hold up merging of this PR, I just wanted to make you aware of this :-)